Resources/Prototypes/_NF/Entities/Structures/Specific/TradingCrates/base.yml

```
@@ -1,5 +1,5 @@
 - type: entity
-  parent: [BaseStructureLockImmuneToEmag, BaseStructureAccessReaderImmuneToEmag, BaseStructure]
+  parent: [BaseStructureLockImmuneToEmag, BaseStructureAccessReaderImmuneToEmag, BaseStructureDynamic]
```

```suggestion
  parent: BaseStructureDynamic
```
No lock needed, nothing to open.  Should fix those issues you mentioned.

---

Tested with whatstone's solution, crates do work as desired: movable, wrenchable, not openable